src/test/java/myapp/HelperBuddy/ToDoTest.java

```diff
@@ -0,0 +1,29 @@
+package myapp.helperbuddy;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ToDoTest {
+    @Test
+    public void testParsing() {
+        String taskData = "T | 1 | evening workout";
+        String[] parts = taskData.split(" \\| ");
+        String description = parts[2].trim();
+        ToDo expectedTask = new ToDo(description);
+        ToDo actualTask = ToDo.parseTask(taskData);
+        assertEquals(expectedTask.getDescription(), actualTask.getDescription());
+    }
+
+    @Test
+    public void testFormatting() {
+        String taskData = "T | 1 | evening workout";
+        String[] parts = taskData.split(" \\| ");
+        String description = parts[2].trim();
+        ToDo task = new ToDo(description);
+        task.markDone();
+        String expected = "T | " + (task.getDone() ? "1" : "0") + " | " + task.getDescription();
+        String actual = "T | 1 | evening workout";
+        assertEquals(expected, actual);
```

Comment:
There are a lot of similarities between testFormating and testParsing, maybe can combine into one function to avoid repetition?